src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,50 @@
+package duke;
+
+import duke.exceptions.InvalidFileTypeException;
+
+import java.io.*;
```

Comment:
Is this import statement too general? Maybe can import specific classes instead :relaxed: